liberum_core/src/node/store.rs

```diff
@@ -27,12 +30,17 @@ pub enum NodeStoreError {
     NodeDoesNotExist { name: String },
     #[error("other error, name: {name}, err: {err}")]
     OtherError { name: String, err: anyhow::Error },
+    #[error("any error, err: {err}")]
+    Any {
```

Comment:
That won't help with the error handling...

Reply:
Fixed.

Reply:
https://github.com/Liberum-Neto/liberum-neto/commit/b8a0902ccf38d9d357c77bf6730cb027c9e6c7d4